README.md

```diff
@@ -11,4 +11,16 @@ Want to add yourself to this list? Do the following:
 * Fork this repository
 * Copy `person-template.md` into the `people` folder, so you have `people/your-name.md
 * Send a pull request
-* Voila!
\ No newline at end of file
+* Voila!
+
+Community
+=========
+
+Our community is spread across various services and maintained by various people. If you've got any questions about those services, please reach out to those people who can help.
+
+* GitHub (here) - [Administrators](https://github.com/orgs/opensourcedesign/teams/core)
+* Twitter - (https://twitter.com/opensrcdesign) - administered by @jancborchardt.
+* Facebook - (https://www.facebook.com/opensrcdesign/) - administered by @razetime.
+* Medium - (https://medium.com/open-source-design) - administered by @razetime, @plastelina, @rdbarlett
```

Comment:
:) actually it needs to be tested in order for the punctuation to not interfere with the user name. Maybe there is no problem. Sorry for the noise.